test/StreamTests.hs

```diff
@@ -141,8 +144,11 @@ simpleWorkbookRow :: Xlsx
 simpleWorkbookRow = def & atSheet "Sheet1" ?~ sheet
   where
     sheet = toWs [((1,1), a1), ((2,1), cellValue ?~ CellText "text at A2 Sheet1" $ def)]
+--    sheets = [("Sheet1" , toWs
+--      [ ((RowIndex 1, ColumnIndex 1), a1)
+--      , ((RowIndex 2, ColumnIndex 1), cellValue ?~ CellText "text at A2 Sheet1" $ def)])]
```

Comment:
why do we have these lines and also a similar  commeted out `sheets`  below?

Reply:
Thanks for spotting - I must have left this in after rebasing